contracts/StakeCurate.sol

```diff
@@ -197,53 +196,54 @@ contract StakeCurate is IArbitrable, IEvidence {
 
   /**
    * @dev Creates a list. They store all settings related to the dispute, stake, etc.
-   * @param _governor The address of the governor.
+   * @param _governorId The id of the governor.
    * @param _requiredStake The Cint32 version of the required stake per item.
    * @param _removalPeriod The amount of seconds an item needs to go through removal period to be removed.
    * @param _arbitratorExtraDataId Id of the internally stored arbitrator extra data
    * @param _metaEvidence IPFS uri of metaEvidence
    */
   function createList(
-    address _governor,
+    uint64 _governorId,
     uint32 _requiredStake,
     uint32 _removalPeriod,
-    uint32 _arbitratorExtraDataId,
+    uint64 _arbitratorExtraDataId,
     string calldata _metaEvidence
   ) external {
+    //require(accounts[_governorId].wallet == msg.sender, "Governor must exist and be the function caller");
```

Comment:
`_governorId < accountCount`
Governor must not be the function caller, you made that up